File: pset6/mario.py
import cs50

print("Height: " , end="")
n = cs50.get_int()

while n<0 or n>23:
    print("Height: " , end="")
    n = cs50.get_int()

for i in range(n):
    for j in range(n-1-i):
        print(" " , end="")
    for k in range(1+i):
        print("#" , end="")
    print(" " , end="")
    for k in range(1+i):
        print("#" , end="")
    print()    
        

File: mario-more/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    printf("Height: ");
    int n = get_int();
    while(n<0 || n>23)
    {
        printf("Height: ");
        n = get_int();
    }
    
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-1-i;j++)
        {
            printf(" ");
        }
        for(int k=0;k<1+i;k++)
        {
            printf("#");
        }
        printf("  ");
        for(int k=0;k<1+i;k++)
        {
            printf("#");
        }
        printf("\n");
    }
}